x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_row_renderer.tsx

```diff
@@ -61,6 +56,48 @@ const SuricataRow = styled.div`
   }
 `;
 
+const SuricataSignature = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top 10px;
+`;
+
+const ButtonContainer = styled.div``;
```

Comment:
Do you want this as an empty? Why not just use a regular div? If we turned the linter back on for styled it would complain loudly about this.

Reply:
I felt like `ButtonContainer` had a nice "semantic" flow when you read it:
```
          <SuricataSignature>
            <ButtonContainer>
              <EuiButton fill size="s" href={createLinkWithSignature(signature)} target="_blank">
                {signature}
              </EuiButton>
            </ButtonContainer>
            <Details>
            // ...
```

vs

```
          <SuricataSignature>
            <div>
              <EuiButton fill size="s" href={createLinkWithSignature(signature)} target="_blank">
                {signature}
              </EuiButton>
            </div>
            <Details>
            // ...
```

I'm 😎 either way, but out of curiosity, which one looks better to you when they are paired like the above?

Reply:
It reads better but it does create a ReactComponent behind the scenes that isn't doing anything which adds a bit more overhead. I'm indifferent but it just is very odd to me to add semantic readability but when I open the actual DOM to trouble shoot I'm greeted with a normal `div` anyways.

Also, if we pull the linter back in, the linter will force us to remove empty or turn off the rule for empty's.

My 2 cents

Reply:
Well, problem solved: That container was an artifact of an earlier, more complicated layout. I removed it.